### PR TITLE
Completed - Issue#1: User can create an entry - Khanh Nguyen

### PR DESCRIPTION
**Pull request by khanhnguyenq**
https://github.com/khanhnguyenq/code-journal/assets/114459075/03d170bf-96ce-456b-ab81-0b54806fdf27

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -6,3 +6,14 @@ let data = {
...
 };
+
+const previousDataJSON = localStorage.getItem('data-object');
+
+window.addEventListener('beforeunload', function (event) {
+  const dataJSON = JSON.stringify(data);
+  localStorage.setItem('data-object', dataJSON);
+});
+
+if (localStorage.getItem('data-object') !== null) {
```
Comment (shawnkost) [inline js/data.js]: You already have a variable for this, so you should just use `previousDataJSON` instead

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here Khanh!

File: js/data.js (hunk 1 above)
Comment (khanhnguyenq) [reply js/data.js]: Thanks Shawn. Updated line 17 on data.js to use variable

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**